### PR TITLE
refactor(CSS):refactor CSS snapshot to a new SDK 

### LINKED ISSUE #1495: [CSS] refactor CSS snapshot to a new SDK
### Proposal

 refactor CSS snapshot to a new SDK,update document

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
refactor CSS snapshot to a new SDK 

**Which issue this PR fixes**:
fixes #1495

## PR Checklist

* [X] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/css' TESTARGS='-run=TestAccCssSnapshot_basic'
==> Checking that code complies with gofmt requirements...

TF_ACC=1 go test ./huaweicloud/services/acceptance/css -v -run=TestAccCssSnapshot_basic -timeout 360m -parallel 4
=== RUN   TestAccCssSnapshot_basic
=== PAUSE TestAccCssSnapshot_basic
=== CONT  TestAccCssSnapshot_basic
--- PASS: TestAccCssSnapshot_basic (1059.90s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/css       1060.030s
```

### CODE REVIEW

**Comment by hmbig2** [comment]
please review after RP:  feat(CSS): refactor CSS cluster resources to new sdk

File: huaweicloud/services/acceptance/css/resource_huaweicloud_css_snapshot_test.go [hunk 1]
```diff
@@ -98,42 +99,15 @@ func testAccCheckCssSnapshotExists() resource.TestCheckFunc {
...
-  }
-
-  backup_strategy {
-    start_time = "00:00 GMT+03:00"
-    prefix     = "snapshot"
-    keep_days  = 14
-  }
-}
+	clusterName := acceptance.RandomAccResourceName()
+	clusterString := testAccCssCluster_basic(clusterName, 1, 1, "tag")
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/css/resource_huaweicloud_css_snapshot_test.go:115]: `basic` method maybe update in the feature, suggest use existing `base` method or define a new one.

File: huaweicloud/services/css/resource_huaweicloud_css_snapshot.go [hunk 2]
```diff
@@ -111,15 +114,17 @@ func resourceCssSnapshotCreate(d *schema.ResourceData, meta interface{}) error {
 
 func resourceCssSnapshotRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*config.Config)
-	cssClient, err := config.CssV1Client(GetRegion(d, config))
+	region := config.GetRegion(d)
+	cssClient, err := config.CssV1Client(region)
+
 	if err != nil {
-		return fmtp.Errorf("Error creating HuaweiCloud CSS client: %s", err)
+		return fmtp.Errorf("Error creating CSS client: %s", err)
```
Comment (Lance52259) [inline huaweicloud/services/css/resource_huaweicloud_css_snapshot.go]: Meaningless modification

File: huaweicloud/services/acceptance/css/resource_huaweicloud_css_snapshot_test.go (hunk 1 above)
Comment (hmbig2) [reply huaweicloud/services/acceptance/css/resource_huaweicloud_css_snapshot_test.go:115]: this resource depends on css cluster resource .  so 
this resource depends on css cluster resource. so if css cluster test changes, it needs to be changed here too

File: huaweicloud/services/css/resource_huaweicloud_css_snapshot.go (hunk 2 above)
Comment (hmbig2) [reply huaweicloud/services/css/resource_huaweicloud_css_snapshot.go]: OK

**Pull request merged**